tools/mmp/driver.cs

```diff
@@ -484,6 +487,8 @@ static void Pack (IList<string> unprocessed)
 
 			if (registrar == RegistrarMode.Default)
 				registrar = RegistrarMode.Dynamic;
+			if (is_extension)
+				registrar = RegistrarMode.Static;
 
```

Comment:
With your suggested replacement if you had a build that wasn't an extension and had the registrar set to static or IL it'd get wiped out and set back to dynamic.